README.md

```diff
@@ -2,28 +2,36 @@
 
 ![Screenshot](./img/qgis-planet.webp)
 
-By Tim Sutton and QGIS Contributors.
+This project aggregates various RSS feeds related to the project, providing a centralized hub for updates and news.
+
+It has been mainly developed by Tim Sutton, Lova Andriarimalala and contributors.
```

Comment:
Probably nicer to credit Lova first since he is the principal author of this incarnation